### ISSUE TITLE
Build fails: error: use of undeclared identifier 'get_current_dir_name'

### ISSUE THREAD

**Issue by yurivict** [DISTRHO/DPF-Plugins#6]
```
src/Window.cpp:1154:36: error: use of undeclared identifier 'get_current_dir_name'
        if (char* const dir_name = get_current_dir_name())
                                   ^
```

FreeBSD 11.2

**Comment by falkTX** [comment]
Please file this report again DPF if possible, problem comes from there.
Patches welcomes, even if against this repo directly.
Though the file browser is still quite experimental, breaking the build is not nice, I know. Sorry for that.

**Comment by falkTX** [comment]
Fixed in DPF and in this repo in e2d9419006ced4419251f379703b517c40ae8dce
